CPC-CHARTER.md

```diff
@@ -156,6 +156,10 @@ ensuring:
       substantial changes before final approval by CPC.
    1. Voting members are responsible for approving funds for budgets delegated
       to the project.
+   1. Voting members are responsible for approving new `collaboration spaces` within the scope
+      for OpenJS Foundation set by the Governing Board, as outlined in the OpenJS Collaboration Network process. 
```

Comment:
`within the scope for` reads a bit odd. Is it necessary to be explicit there? Could it be simplified?

```suggestion
   1. Voting members are responsible for approving new `collaboration spaces` as outlined in the OpenJS 
   Collaboration Network process. 
```